viewing_party/party.py

```diff
@@ -0,0 +1,141 @@
+## TEST WAVE 1 ##
+
+def create_movie(movie_title, genre, rating):
+    movie = {}
+    if movie_title and genre and rating:
+        movie["title"] = movie_title
+        movie["genre"] = genre
+        movie["rating"] = rating
+        return movie
```

Comment:
I generally recommend making dictionaries in place if you don't plan to further modify them:

```suggestion
    if movie_title and genre and rating:
        movie = {
            "title": movie_title,
            "genre": genre,
            "rating": rating
        }
        return movie
```